### DISCUSSION TITLE
What is being done towards reducing the amount of issues?

### DISCUSSION THREAD

**Discussion by ffMathy** [oven-sh/bun#9166] [Q&A]
Looking at the insights, it seems like more issues are popping up than are being closed. Perhaps this is natural. Just curious if the Bun team has enough contributors and funding to be able to get on top of it over time?

**Comment by truongan07** [comment]
I hope : 
- Google & Amazon donate Node
- Meta & Twitter donate Deno
- Apple & Microsoft donate Bun

**Comment by matoruru** [comment]
Bun's org/repos should enable GitHub sponsor button!

**Comment by Jarred-Sumner** [comment]
We are well-funded, but we don't have enough engineers working on Bun right now.

We are hiring systems engineers in San Francisco - https://bun.sh/careers
